src/components/list-details/list-details.tsx

```diff
@@ -38,7 +38,9 @@ const ListDetails: FC<ListDetailsProps> = ({
           )}
           {typeof labels !== "string" &&
             labels.map((label: string) => (
-              <p className="text-small-caption">{label}</p>
+              <p key={label} className="text-small-caption">
+                {label}
+              </p>
```

Comment:
Why is this change added? It does not seem to be relevant to the change at hand.

I cannot tell from the commit comment either.

Reply:
This was to remove an error message from the console